common.blocks/button/button.bemhtml

```diff
@@ -1,4 +1,11 @@
 block('button')(
+    def()(function() {
+        var isRealButton = (['button', 'input'].indexOf(apply('tag')) >= 0) &&
```

Comment:
> in this case you should change def mode(not tag) in attach too.
I don't understand you have to change anything in `attach`. It [passes button's tag explicitly as a bemjson](https://github.com/bem/bem-components/blob/v2/common.blocks/attach/attach.bemhtml#L13-L17).

> And it's very strange to change tag in def mode

Yes, as well as to call `apply('tag')` twice. At least, my variant should be more performant, I guess.

> agree
so?